client/asset/btc/btc.go

```diff
@@ -557,7 +573,7 @@ type baseWallet struct {
 	useSplitTx        bool
 	useLegacyBalance  bool
 	segwit            bool
-	legacyRawFeeLimit bool
+	legacyRawFeeLimit bool // wut dis?
```

Comment:
Looks only needed at the level of `rpcWallet`, not `ExchangeWallet`.

BTW, in `rpcWallet`, would be nice if `SendRawTransactionLegacy` and `SendRawTransaction` were situated remotely close to the caller `sendRawTransaction`